_locales/en/messages.json

```diff
@@ -646,6 +646,12 @@
     "configurationBatteryScale": {
         "message": "Voltage Scale"
     },
+    "configurationVbatPidCompensation": {
+        "message": "Vbat PID Compensation"
+    },
+    "configurationVbatPidCompensationHelp": {
+        "message": "Increase the PID values to compensate when Vbat gets lower. This will give more constant flight characteristics throughout the flight."
```

Comment:
Sorry for the confusion. I forgot I converted it back to Q12
Op 13 jul. 2016 01:33 schreef "Boris B." <dreambb1982@gmail.com>:

> Actually you are right....that part needs a cleanup. It was the old
> implementation before I converted it to Q12.
>
> I realised min voltage is not needed at all. Once you know the max you can
> scale it down till a hardcoded limit. The scaling goes down to 30% from the
> top of my head.
> Op 13 jul. 2016 01:27 schreef "Boris B." <dreambb1982@gmail.com>:
>
>>
>> https://github.com/betaflight/betaflight/blob/master/src/main/flight/mixer.c#L763
>> Op 13 jul. 2016 01:25 schreef "Michael Keller" <notifications@github.com
>> >:
>>
>>> In _locales/en/messages.json
>>> <https://github.com/betaflight/betaflight-configurator/pull/111#discussion_r70540910>
>>> :
>>>
>>> > @@ -646,6 +646,12 @@
>>> >      "configurationBatteryScale": {
>>> >          "message": "Voltage Scale"
>>> >      },
>>> > +    "configurationVbatPidCompensation": {
>>> > +        "message": "Vbat PID Compensation"
>>> > +    },
>>> > +    "configurationVbatPidCompensationHelp": {
>>> > +        "message": "Increase the PID values to compensate when Vbat gets lower. This will give more constant flight characteristics throughout the flight."
>>>
>>> I can't find a reference to calculateBatteryPercentage. I must really
>>> be seeing things. :-(
>>>
>>> ~/git/betaflight$ grep calculateBatteryPercentage src/main/flight/mixer.c
>>> ~/git/betaflight$
>>>
>>> —
>>> You are receiving this because you commented.
>>> Reply to this email directly, view it on GitHub
>>> <https://github.com/betaflight/betaflight-configurator/pull/111/files/c63af70999b71270e4c2c6b4d60295356c402a65#r70540910>,
>>> or mute the thread
>>> <https://github.com/notifications/unsubscribe/AKQlhM4RXFy_2f70G4NHtrkQ40jvOpRYks5qVCJrgaJpZM4JKUzM>
>>> .
>>>
>>